dbms/src/Flash/ResourceControl/LocalAdmissionController.h

```diff
@@ -416,8 +423,11 @@ class LocalAdmissionController final : private boost::noncopyable
         }
     }
 
-    std::optional<uint64_t> getPriority(const std::string & name)
+    std::optional<uint64_t> getPriority(const std::string & name, bool enable_resource_control = true)
```

Comment:
why modify here?

Reply:
Just make all public member function same.

Reply:
It seems that LAC’s public function does not need `bool enable_resource_control`.
If enable_resource_control = false, the function of LAC will not be called.